src/src/tx_main.cpp

```diff
@@ -954,6 +959,13 @@ void loop()
 {
   uint32_t now = millis();
 
+  #if defined(PLATFORM_ESP32) && defined(USE_BLE_JOYSTICK)
```

Comment:
Maybe should be aligned to other similar checks... With or without `PLATFORM_ESP32` define...